### ISSUE TITLE
3D Print Details

### ISSUE THREAD

**Issue by snowake4me** [bluebottech/BBT-LED-Smart-Lamp#1]
First and foremost - thanks for this excellent project!  As an addressable LED _and_ 3D printing enthusiast, it hits right in my sweetspot. :)

I've got all my parts on hand and ready to assemble - but I'm having some trouble with the 3D prints.  My first attempt at printing the base ran for several hours before running afoul and turning to scribbles:  A second attempt, with supports, finally completed after 20+ hours, and with MUCH effort, I have managed to remove the support material.

However - my slicer (Cura 4.11.0) complains when I import the 'glass' STL file, saying "Model Errors:  The highlighted areas indicate either missing or extraneous surfaces.  Fix your model and open it again..."

I've tried opening the STL mesh in Fusion 360, but lack the skills / experience to identify and resolve the issue.

Finally - did you print this piece (the 'glass') horizontally?  Feels like the only way it will fit on my Ender 3, but - OH the support material!

Thanks again for this great design and DIY project.  Looking forward to complete for my daughter's upcoming birthday....

**Comment by bluebottech** [comment]
The model itself definitely has some jank to it so Cura might complain, but
it still lets me slice and print it as is without modification.

Here is how I printed the two main pieces. It does take a ton in support
material unfortunately. Hope this helps!

[image: image.png]

[image: image.png]

**Comment by snowake4me** [comment]
Thanks for the response, @bluebottech !  Unfortunately, I can't seem to view the above attached images?  I'll keep trying.

At any rate - I finally just went ahead and printed the "glass" with my transparent filament and supports enabled.  It took over 2.5 days, and used almost half a spool of filament (453g, IIRC) - but in the end, it completed successfully.  I've been able to remove the supports from the outside edges - but as for that huge / dense column that runs all the way through the middle of the print?  I'm not sure HOW I'm going to get that out without destroying the whole thing.  I'll try and post some pictures of my print, somewhere - as it really IS an impressive 'build', even if I'm not able to salvage it.

This morning, after pulling the 3-day print off the printer:  I sliced and kicked off another on with "no supports". Last I checked on it, it was still doing okay - but I'm scared of what will happen when it gets to the top half and tries to close off the circle. :). Figured it was worth a shot... :)

Thanks again, and have a GREAT weekend!  I've still got a few days to try to get this build together - so I just might make it.  Peace!

**Comment by bluebottech** [comment]
yeah it's a tough print for sure. I usually have to use needle nose pliers
or wire snipers to get in and carefully rip some of the filament out. Glad
you are making progress!
